package.json

```diff
@@ -34,8 +34,8 @@
         "socket.io-client": "^4.1.2"
     },
     "peerDependencies": {
-        "react": ">=16",
-        "next": "10.x"
+        "react": ">=17",
+        "next": "^11.0.0"
```

Comment:
@JugglerX we have existing projects that use sourcebit and are running under next 10.
So I suggest making it ">=10"

Reply:
```suggestion
        "next": ">=10"
```